README.md

```diff
@@ -1,9 +1,11 @@
-# go-execplus
+# go-execplus  [![Build Status](https://travis-ci.org/Originate/go-execplus.svg?branch=master)](https://travis-ci.org/Originate/go-execplus)
```

Comment:
Can we have the link go to `https://travis-ci.org/Originate/go-execplus/branches`? I don't like that clicking the link by default shows you latest build (which could be any branch). Travis doesn't appear to have a way to just see the build for a given branch. But the branches page should have the default branch listed at the top.